exercises/concept/meltdown-mitigation/.meta/exemplar.py

```diff
@@ -67,14 +60,11 @@ def fail_safe(temperature, neutrons_produced_per_second, threshold):
     - `temperature * neutron per second` is not in the above-stated ranges ==  'DANGER'
     """
     output = temperature * neutrons_produced_per_second
-    operational_percentage = int((output / threshold) * 100)
-    safety_range = 'UNKNOWN'
+    operational_percentage = int(output / threshold) * 100
```

Comment:
If you haven't found why tests failed?
> `int(output / threshold) * 100`
Above expression returns 0, when output is less than threshhold, which is not expected behavior.
This must be changed to `int((output / threshold) * 100)`, to behave correctly...